src/charm.py

```diff
@@ -44,6 +46,19 @@ def __init__(self, *args):
         self.tls = KafkaTLS(self)
         self.provider = KafkaProvider(self)
         self.restart = RollingOpsManager(self, relation="restart", callback=self._restart)
+        self.grafana_dashboards = GrafanaDashboardProvider(self)
+        self.metrics_endpoint = MetricsEndpointProvider(
+            self,
+            jobs=[
+                {
+                    "static_configs": [
+                        {
+                            "targets": ["*:9100"],
+                        }
+                    ]
+                }
+            ],
+        )
```

Comment:
Could we 1 line this and add a lint exception?

Reply:
I can change this to be better without the exception